src/gui/previewselectdialog.h

```diff
@@ -54,19 +55,30 @@ class PreviewSelect : public QDialog, private Ui::preview
         NB_COLUMNS
     };
 
-    PreviewSelect(QWidget* parent, BitTorrent::TorrentHandle *const torrent);
-    ~PreviewSelect();
+    PreviewSelectDialog(QWidget *parent, BitTorrent::TorrentHandle *const torrent);
+    ~PreviewSelectDialog();
 
 signals:
     void readyToPreviewFile(QString) const;
 
+protected:
+    void showEvent(QShowEvent *event) override;
+
 private slots:
     void previewButtonClicked();
 
 private:
+    void loadWindowState();
+    void saveWindowState();
```

Comment:
I also renamed this two methods from loadSettings(), it follows names which are used in optionsdlg